recipes/newrelic/cassandra/debian.yml

```diff
@@ -37,16 +37,16 @@ validationNrql: "SELECT count(*) from CassandraSample where hostname like '{{.HO
 
 inputVars:
   - name: "NR_CLI_DB_USERNAME"
-    prompt: "Enter the username to connect to Cassandra via JMX."
+    prompt: "Cassandra connection Username (via JMX)"
   - name: "NR_CLI_DB_PASSWORD"
-    prompt: "Enter the password to connect to Cassandra via JMX."
+    prompt: "Cassandra connection Password (via JMX)"
     secret: true
   - name: "NR_CLI_DB_HOSTNAME"
-    prompt: "Confirm default or enter new value for the Cassandra node hostname."
+    prompt: "Cassandra node Hostname (default: localhost)"
     default: "localhost"
   - name: "NR_CLI_DB_PORT"
-    prompt: "Confirm default port or enter new value. The port where Cassandra is listening and exposing metrics and variables via JMX."
-    default: 7199
```

Comment:
7199 should be the correct value. This is also what we document https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/cassandra-monitoring-integration